editor/utils/formHelpers.ts

```diff
@@ -187,3 +187,28 @@ export const handleFormValueOutput = (
   // handle everything else as a nulllable string
   return trimStringNullable(value);
 };
+
+/**
+ * Validate a lat/lon coordinate and round it to the
+ * MAP_COORDINATE_PRECISION.
+ *
+ * Returns null if value is NaN or if the number is
+ * is not within the allowed latitude or longitude range
+ */
+export const handleCoordinate = (
+  value: number | null,
+  isLatitude: boolean
+): number | null => {
+  if (value === null || isNaN(value)) {
+    return null;
+  }
+  if (isLatitude && Math.abs(value) > 90) {
+    return null;
+  } else if (!isLatitude && Math.abs(value) > 180) {
+    return null;
+  }
+  return (
+    Math.round(value * Math.pow(10, MAP_COORDINATE_PRECISION)) /
+    Math.pow(10, MAP_COORDINATE_PRECISION)
+  );
```

Comment:
I like how you did this with the `Math` built in library. I was really surprised to see that there isn't a rounding method that takes an argument to define precision. 

I'm not sure this is good advice, but when looking for something that did offer that precision argument, the `.toFixed()` ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed)) method on the the number class. (number prototype? I'm not sure what the fundamental are called here.) `.toFixed()` might offer slightly improved readability, but, again, this is indubitably correct how you've done it.